docs/grammar/SolidityParser.g4

```diff
@@ -482,7 +482,7 @@ forStatement: For LParen (simpleStatement | Semicolon) (expressionStatement | Se
 whileStatement: While LParen expression RParen statement;
 doWhileStatement: Do statement While LParen expression RParen Semicolon;
 /**
- * A continue statement. Only allowed inside for, while or do-while loops.
+ * A continued statement. Only allowed inside for, while or do-while loops.
```

Comment:
This one too. We have a `continue` keyword in the language and this refers to it. You can actually see it defined below.